Makefile

```diff
@@ -79,6 +79,9 @@ lint: lint-go lint-cli
 
 .PHONY: lint-go
 lint-go:
+ifneq ($(shell cat .go-version),$(shell go list -m -f '{{.GoVersion}}'))
+	$(error The go versions in go.mod and .go-version must match)
+endif
```

Comment:
Make a note that when ServiceBot does this for us, we can remove this check?

Reply:
I don't think we want to remove it. The problem this is designed to block is someone making a manual update to the go.mod version without also changing the .go-version.

If the service bot updates both at once, there's no issue.